api/Window.json

```diff
@@ -7564,19 +7568,23 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": true
+              "version_added": true,
+              "notes": "Firefox emits a <code>resize</code> event on page load, but it will soon stop doing so (see <a href='https://bugzil.la/1528052'>bug 1528052</a>)."
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": true,
+              "notes": "Firefox emits a <code>resize</code> event on page load, but it will soon stop doing so (see <a href='https://bugzil.la/1528052'>bug 1528052</a>)."
             },
             "ie": {
               "version_added": true
             },
             "opera": {
-              "version_added": "32"
+              "version_added": "32",
+              "notes": "Opera does not emit a <code>resize</code> event on page load."
```

Comment:
I don't know what the convention here is, but I try to avoid naming browsers in notes, since that is clear from the context, and this stops note text from being the same, and that can be helpful to code that renders tables. So I would say just "The \<code>\<resize>\</code> event is not emitted on page load".

Would happily hear from @Elchi3  on this though.

Reply:
We've included the browser names in notes on several other features, especially when including a version number.  I don't really think we have a convention one way or the other.

Reply:
Yes, fair enough!

Reply:
Personally, I'm in favor of keeping the browser name in the notes.  It helps maintain context if the note is quoted anywhere else.  Perhaps we should open a discussion and define a specific convention?